libraries/Microsoft.Bot.Builder/InvokeResponse.cs

```diff
@@ -26,5 +26,14 @@ public class InvokeResponse
         /// will have a body generated by JSON serializing the object in this field.
         /// </remarks>
         public object Body { get; set; }
+
+        /// <summary>
+        /// Gets a value indicating whether the invoke response was successful.
+        /// </summary>
+        /// <value>
+        /// A value that indicates if the HTTP response was successful.
+        /// true if <see cref="Status"/> was in the Successful range (200-299); otherwise false.
+        /// </value>
+        public bool IsSuccessStatusCode => Status >= 200 && Status <= 299;
```

Comment:
does this need to be serialized?  I would make this a method if this is truly needed.
You know that HttpResponseMessage has .IsStatusSuccessCode() ?

Reply:
Ah, good point, I'll make it a method, we do use the >= 200 && <=299 all over the place and this would make the checks less verbose, I created this based on the [HttpResponseMessage.IsSuccessStatusCode](https://docs.microsoft.com/en-us/uwp/api/windows.web.http.httpresponsemessage.issuccessstatuscode?view=winrt-18362)